### PR TITLE
Use batched nodary endpoint

### PR DESCRIPTION
**Pull request by metobom**
Closes https://github.com/api3dao/tasks/issues/720

### CODE REVIEW

File: data/apis/nodary/api-data.json [hunk 1]
```diff
@@ -5,171 +5,177 @@
...
+        "TSLA/USD",
+        "TSM/USD",
+        "PYPL/USD",
+        "BABA/USD",
+        "QQQ/USD",
+        "DIS/USD",
+        "INTC/USD"
+      ],
+      [
+        "cbETH/ETH Exchange Rate",
```
Comment (bdrhn9) [inline data/apis/nodary/api-data.json:30]: Is there any meaning of current sorting? If not, I'd like to see them sorted by name.

File: data/apis/nodary/oises/nodary.json [hunk 2]
```diff
@@ -75,12 +75,12 @@
...
...*\" } },\n    \"GME/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"GOOGL/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"INTC/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"JPM/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"META/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"MSFT/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"NFLX/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"NVDA/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"PYPL/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"QQQ/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"SPY/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"TSLA/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n    \"TSM/USD\": { path: \"feed/latestV2\", parameters: { name: \"*\" } },\n  }\n  return {\n    endpointParameters: {\n      path: preProcessingObject[endpointParameters.name].path,\n      ...preProcessingObject[endpointParameters.name].parameters,\n    },\n  }\n}\n",
```
Comment (bdrhn9) [inline data/apis/nodary/oises/nodary.json:78]: `{ path: \"feed/latestV2\", parameters: { name: \"*\" } } `

There is no parameter named `name` in that endpoint. `names` exist and it doesn't take `*` as a value. Expected behavior can be achieved just not setting `names` parameter. Also it requires the fix in `endpoints` section of the OIS.

File: data/apis/nodary/api-data.json (hunk 1 above)
Comment (metobom) [reply data/apis/nodary/api-data.json:30]: Data feeds are batched by category, also if a category has more than 100 feeds, it is separated into two batches.

Comment (bdrhn9) [reply data/apis/nodary/api-data.json:30]: I see. Could you clarify why the feed names aren't sorted by name within these batches?

**Pull request closed without merging**